### PR TITLE
Integrate new Footer designs

### LINKED ISSUE #8421: Refactor Footer
The current footer is very busy and very long on mobile and could use some TLC on padding/borders being uneven. The green icons are also raising questions. Gareth suggested a [new footer](https://www.sketch.com/s/6ced6ea4-e7c2-4a04-a4b6-6b77a531e35c) as part of some exploration he'd done a few months back, for a cleaner, streamlined footer.

Start with the one without the newsletter, then create a new ticket for the one that has it.

### PR DESCRIPTION
**Pull request by rcantin-w**
## Who is this for?
Users/New design

## What is it doing for them?
Makes the footer less busy, matches it with the newest designs. This is phase one, as in phase two we'll look at moving the newsletter signup in as well.

Closes #8421

### CODE REVIEW

**Comment by rcantin-w** [comment]
> Size Change: +6.36 kB (+1%)

Why 😬

**Comment by jamieparkinson** [comment]
@rcantin-w not totally sure about this but I think it's possible this is a false alarm, in that the increase might be that code which was previously common has become duplicated over pages (ie no increase for any individual page) - as previously, could be worth seeing what the `yarn build` output tells us on this branch vs `main`?

**Comment by rcantin-w** [comment]
@jamieparkinson you might be right? Main seems to output heavier pages than the footer branch, and I haven't moved anything from common 🤔 

Main
main

Footer
footer

**Comment by rcantin-w** [comment]
> ah, the magic of Webpack - good outcome though!

Reckon we can consider it a non-issue then?

File: common/views/components/FindUs/FindUs.tsx [hunk 1]
```diff
@@ -1,75 +1,36 @@
-import styled from 'styled-components';
-import { font } from '../../../utils/classnames';
+import { FunctionComponent } from 'react';
```
Comment (davidpmccormick) [inline common/views/components/FindUs/FindUs.tsx:1]: nit: prefer `FC` (although I realise you didn't add this)

File: common/views/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -1,346 +1,237 @@
...
+const FooterNavigationContainer = styled(FooterBasicSection)`
   display: flex;
+  flex-wrap: wrap;
   justify-content: space-between;
-  border-top: 1px solid ${props => props.theme.color('neutral.700')};
 `;
 
-const FooterBottom = styled(Space).attrs({
-  v: { size: 'xl', properties: ['padding-bottom'] },
+// TODO mismatched links, what do
```
[the remark this answers is not in the document]
Comment (rcantin-w) [reply common/views/components/Footer/Footer.tsx:46]: Ahh yeah thanks for catching this, I actually had this to figure out still... Comment from the ticket:
> Moving forward we were making the links underlined with a hover behaviour that removed the underline. This shows it doesn't work for everything as it doesn't look great/natural on the Address & Phone number. I've kept them on the old behaviour (not underline, green on hover), but am not loving the mix. Maybe something to review later on...

File: common/views/components/FindUs/FindUs.tsx [hunk 3]
```diff
@@ -1,75 +1,36 @@
...
-            {wellcomeCollectionAddress.streetAddress}
-          </span>
-          {wellcomeCollectionAddress.addressLocality}{' '}
-          {wellcomeCollectionAddress.postalCode}
-          <span className="block">
-            {wellcomeCollectionAddress.addressCountry}
-          </span>
-        </Space>
-      </span>
+      <Space v={{ size: 'm', properties: ['margin-bottom'] }} as="p">
```
[the remark this answers is not in the document]
Comment (rcantin-w) [reply common/views/components/FindUs/FindUs.tsx]: Thanks, I've addressed!

File: common/views/components/Footer/Footer.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (rcantin-w) [reply common/views/components/Footer/Footer.tsx]: As discussed will leave as is for now

File: common/views/components/FindUs/FindUs.tsx [hunk 4]
```diff
@@ -1,75 +1,36 @@
...
-      }}
-      as="p"
-      className="block"
-    >
-      <a href={`/pages/${prismicPageIds.gettingHere}`}>Getting here</a>
+const FindUs: FC = () => (
+  <>
+    <Space v={{ size: 'm', properties: ['margin-bottom'] }} as="p">
+      <a href={wellcomeCollectionAddress.addressMap} className="plain-link">
+        <span className="block">{wellcomeCollectionAddress.streetAddress}</span>
```
Comment (gestchild) [inline common/views/components/FindUs/FindUs.tsx:13]: Did we decide whether we were moving away from these utility classes?

Comment (rcantin-w) [reply common/views/components/FindUs/FindUs.tsx:13]: We did, good shout.

**Pull request merged**